book/src/adding_board_support.md

```diff
@@ -0,0 +1,117 @@
+# Adding Support for a Board
+
+This document serves as a guide as to what is currently needed for adding support
+for a board/device to Ariel OS.
+
+Feel free to report anything that is unclear or missing!
+
+## Adding Support for a Board
+
+The more similar a board is to one that is already supported, the easier.
+It is usually best to copy and adapt an existing one.
+
```

Comment:
Clarify that this needs forking `ariel-os/ariel-os` and work in there, currently?